website/docs/r/lb_nat_rule.html.markdown

```diff
@@ -65,6 +64,7 @@ The following arguments are supported:
 * `idle_timeout_in_minutes` - (Optional) Specifies the idle timeout in minutes for TCP connections. Valid values are between `4` and `30` minutes. Defaults to `4` minutes.
 * `enable_floating_ip` - (Optional) Are the Floating IPs enabled for this Load Balncer Rule? A "floating” IP is reassigned to a secondary server in case the primary server fails. Required to configure a SQL AlwaysOn Availability Group. Defaults to `false`.
 * `enable_tcp_reset` - (Optional) Is TCP Reset enabled for this Load Balancer Rule? Defaults to `false`.
+* `resource_group_name` - (Optional / **Deprecated**) The name of the resource group in which to create the resource.
```

Comment:
which means this is no longer forcenew

Reply:
Here is using [SchemaResourceGroupNameDeprecated() ](https://github.com/terraform-providers/terraform-provider-azurerm/blob/326ae0569f803b4fbaee7d3a40aaa56ee6568ce8/azurerm/helpers/azure/resource_group.go#L21)which doesn't include forcenew.